packages/react-diagrams-routing/package.json

```diff
@@ -41,5 +41,8 @@
 		"paths-js": "^0.4.9",
 		"react": "16.*"
 	},
+	"devDependencies": {
+		"rimraf": "^3.0.2"
```

Comment:
rather move this to the root package.json :)

Reply:
You are right.

I tried initially to use the root one, but it was incorrectly done, that' why I though it had to be in the packages.

Thanks.